### PR TITLE
Computed properties vs functions

### PR DESCRIPTION
**Pull request by edmonston**
As discussed with @bryanoltman and @hgoh-blueapron in this PR

https://github.com/blueapron/iphone-application/pull/2208

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -459,6 +459,50 @@ var diameter: Double {
...
+var urlString: String?  {
+  var components = URLComponents()
+  components.scheme = "blueapron"
+  component.host = identifier
+  return components.url?.absoluteString
+}
+```
+
+```swift
+var func toTokens() -> [String]  {
```
Comment (hgoh) [inline README.markdown]: Typo! `var func`

File: README.markdown [hunk 2]
```diff
@@ -459,6 +459,50 @@ var diameter: Double {
...
+func toURL() -> URL {
+  var components = URLComponents()
+  components.scheme = "blueapron"
+  component.host = identifier
+  return components.url?.absoluteString
+}
+```
+
+```swift
+var tokens: [String]  {
```
Comment (hgoh) [inline README.markdown]: Two spaces between `[String]` and `{`!

File: README.markdown [hunk 3]
```diff
@@ -459,6 +459,50 @@ var diameter: Double {
...
+  component.host = identifier
+  return components.url?.absoluteString
+}
+```
+
+```swift
+var func toTokens() -> [String]  {
+    let scanner = Scanner()
+    var tokens = [String]()
+     while !(scanner.isAtEnd) {
```
Comment (hgoh) [inline README.markdown]: Extra space before the `while`!

File: README.markdown [hunk 4]
```diff
@@ -459,6 +459,50 @@ var diameter: Double {
...
+  component.host = identifier
+  return components.url?.absoluteString
+}
+```
+
+```swift
+var tokens: [String]  {
+    let scanner = Scanner()
+    var tokens = [String]()
+     while !(scanner.isAtEnd) {
```
Comment (hgoh) [inline README.markdown]: This one too!

Comment (edmonston) [reply README.markdown]: oops fixing these, you have a great future ahead of you as a human linter

**Review by hgoh — APPROVED** [review]
Beautiful!

**Pull request merged**